include/srdfdom/visibility_control.h

```diff
@@ -0,0 +1,35 @@
+#ifndef SRDFDOM__VISIBILITY_CONTROL_H_
```

Comment:
Should we use [GenerateExportHeader](https://cmake.org/cmake/help/latest/module/GenerateExportHeader.html) .? Do you know what are the drawbacks .? I'm happy to open a PR against your branch

Reply:
There has been some discussion here: https://github.com/ros-planning/moveit2/pull/286#pullrequestreview-496315633, and I think using the visibility headers instead of the cmake function would be better. To quote from links from that discussion:

> In summary, I prefer the boilerplate header we use (e.g. https://github.com/ros2/rclcpp/blob/231b991098d685ff019c44edb80a8ff170d65e4a/rclcpp/include/rclcpp/visibility_control.hpp#L25-L54) to the boilerplate generated code that cmake has to generate every time you build. Neither change, but the static one is nicer in my opinion because you don't waste time generating it each time and because it lets static code analysis (or like autocomplete for text editors) work without having to first build the package and have the build/install space for that package on the path for your tool. Also, if you were to port a package to a different build system (e.g. bazel or something) then you don't have to also port the behavior of `generate_export_header`.